resources/patches/iridium-browser/Remove-EV-certificates.patch

```diff
@@ -13,7 +13,7 @@ Instead, let them appear just like regular ones.
 
```

Comment:
Would this work on Windows? I'm not exactly sure how this addresses #354. (Granted I haven't seen if this code has changed significantly in 66).

Reply:
Would you be able to revert these changes back to the original form? If #354 is still an issue, it'll be a pain to fix it.

Out of curiosity, why did you choose to write the patch this way?

Reply:
I just realized that I used an older version of the patch :confused:  It should be fine now.